Index: domain/okr/okr/ValueObjects.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>const DescriptionValueObject = require(\"./values/description\");\r\nconst IdValueObject = require(\"./values/IdValue\");\r\nconst ObjectiveValueObject = require(\"./values/objective\");\r\nconst ProgressValueObject = require(\"./values/progress\");\r\nconst TitleValueObject = require(\"./values/title\");\r\n\r\nmodule.exports = {\r\n    DescriptionValueObject, \r\n    IdValueObject, \r\n    ObjectiveValueObject, \r\n    ProgressValueObject, \r\n    TitleValueObject}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/domain/okr/okr/ValueObjects.js b/domain/okr/okr/ValueObjects.js
--- a/domain/okr/okr/ValueObjects.js	
+++ b/domain/okr/okr/ValueObjects.js	
@@ -4,9 +4,11 @@
 const ProgressValueObject = require("./values/progress");
 const TitleValueObject = require("./values/title");
 
-module.exports = {
-    DescriptionValueObject, 
-    IdValueObject, 
-    ObjectiveValueObject, 
-    ProgressValueObject, 
-    TitleValueObject}
\ No newline at end of file
+
+module.exports = valueObjects = {
+  DescriptionValueObject,
+  IdValueObject,
+  ObjectiveValueObject,
+  ProgressValueObject,
+  TitleValueObject,
+};
